core/src/services/mysql/config.rs

```diff
@@ -26,11 +26,10 @@ use serde::Serialize;
 #[serde(default)]
 #[non_exhaustive]
 pub struct MysqlConfig {
-    /// This connection string is used to connect to the mysql service. There are url based formats:
+    /// This connection string is used to connect to the mysql service. There are url based formats.
     ///
-    /// ## Url
```

Comment:
Javadoc can hardly handle headings. Update the comment in an identical expressed way.